frontend/pages/milestones/MilestoneListPage.jsx

```diff
@@ -1,5 +1,145 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import Mytable from '@components/common/table';
+// 마일스톤 리스트 테이블
+import service from '@services';
+// 마일스톤 데이터
+import styled from 'styled-components';
+// import MainPageLayout from '@layouts/MainPageLayout';
+import { GoMilestone, GoCheck, GoCalendar } from 'react-icons/go';
 
-const MilestoneListPage = () => <div />;
+const MilestoneListPage = () => {
+  const [open, setOpenMilestone] = useState([]);
+  const [close, setClosedMilestone] = useState([]);
+  const [data, setData] = useState([]);
+  const clickEditBtn = async () => {
+    // 수정버튼
+  };
+  const clickCloseBtn = async () => {
+    // close버튼
+  };
+  const clickDeleteBtn = async () => {
+    // dekete 버튼
+  };
+  const getOpen = async () => {
+    const milestones = await service.getOpenMilestones();
+    setOpenMilestone(milestones.data);
+  };
+  const getClosed = async () => {
+    const milestones = await service.getClosedMilestones();
+    setClosedMilestone(milestones.data);
+  };
+  const getData = async (status) => {
+    const milestones =
+      status === 'close' ? await service.getClosedMilestones() : await service.getOpenMilestones();
+    setData(milestones.data);
+  };
+  useEffect(() => {
+    getOpen();
+    getClosed();
+    getData();
+  }, []);
+  return (
+    <>
+      <Mytable
+        width='100%'
+        header={() => {
+          return (
+            <LabelListHeader>
+              <td colSpan='2'>
+                <HeaderText>
+                  <OpenBtn onClick={() => getData('open')}>
+                    <GoMilestone />
+                    {open.length} Open
+                  </OpenBtn>
+                  <CloseBtn onClick={() => getData('close')}>
+                    <GoCheck />
+                    {close.length} Closed
+                  </CloseBtn>
+                </HeaderText>
+              </td>
+            </LabelListHeader>
+          );
+        }}
+        body={() => {
+          return data.map((milestone) => {
+            return (
+              <TR key={milestone.id}>
+                <TD align='left' padding='10px'>
+                  <Title>{milestone.title}</Title>
+                  <GoCalendar /> Due by {milestone.dueDate.substring(5, 7)},
+                  {milestone.dueDate.substring(8, 10)},{milestone.dueDate.substring(0, 4)}
+                  <br />
```

Comment:
반복적인 객체접근은 먼저 비구조화 할당하면 더 좋을 것 같습니다
```suggestion
        body={() => {
          return data.map((milestone) => {
            const { dueDate } = milestone;
            const year = dueDate.substring(0, 4);
            const month = dueDate.substring(5, 7);
            const day = dueDate.substring(8, 10);
            return (
              <TR key={milestone.id}>
                <TD align='left' padding='10px'>
                  <Title>{milestone.title}</Title>
                  <GoCalendar />{`Due by ${month} ${day}, ${year}`}
                  <br />
```

Reply:
추가로 날짜를 포멧팅하는 유틸함수 하나를 만들어서 사용해도 좋을것같습니다!

Reply:
제가 리스트 컴포넌트 만들면서 react-moment 모듈을 설치했는데 포매팅 관련 기능도 있을 것 같네요